client/my-sites/people/people-section-nav/index.jsx

```diff
@@ -111,13 +111,18 @@ class PeopleSectionNav extends Component {
 				path: '/people/viewers/' + siteFilter,
 				id: 'viewers',
 			},
+			{
+				title: translate( 'Invitations', { context: 'Filter label for invitations list' } ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **9** times:
`translate( 'Invitations', { context: 'Filter label for people list'} )` *ES Score: 24*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/4426_implement_invitees_list)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?